File: LeetCode/problems/1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */

var maximumWealth = function(accounts){
    return Math.max(...accounts.map((el) => el.reduce( (a,b)=>a+b)));
}

var maximumWealth2 = function(accounts) {
    let customers = [];
    let max = 0;
    for(let customer of accounts){
        max = Math.max( customer.reduce(function(a,b){return a+b;},0), max)
    }

    return max;
};

File: LeetCode/problems/1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int highWealth = 0;
        int tempWealth = 0;
        for(int[]a : accounts){
            tempWealth = 0;
            for(int b : a){
                tempWealth+=b;
            }
            if(tempWealth > highWealth){
                highWealth = tempWealth;
            }
        }
        return highWealth;
    }
}